x-pack/plugins/security_solution/server/endpoint/services/agent/clients/lib/types.ts

```diff
@@ -5,8 +5,20 @@
  * 2.0.
  */
 
+import type { Logger } from '@kbn/core/server';
+import type { ActionsClient } from '@kbn/actions-plugin/server';
 import type { AgentStatusRecords } from '../../../../../../common/endpoint/types';
 
+import type { ResponseActionAgentType } from '../../../../../../common/endpoint/service/response_actions/constants';
 export interface AgentStatusClientInterface {
   getAgentStatuses: (agentIds: string[]) => Promise<AgentStatusRecords>;
 }
+
+export interface GetAgentStatusOptions {
+  agentType: ResponseActionAgentType;
+  agentIds: string[];
+  connectorActionsClient: ActionsClient;
+  logger: Logger;
+}
+
+export type AgentType = 'endpoint' | 'sentinel_one' | 'crowdstrike';
```

Comment:
Please use the existing type `ResponseActionAgentType` defined here: https://github.com/elastic/kibana/blob/cd3e684a895427652c9bea635c635905064e0b4c/x-pack/plugins/security_solution/common/endpoint/service/response_actions/constants.ts#L16

Reply:
Right! thank you! :)